packages/uniforms-bridge-simple-schema-2/__tests__/SimpleSchema2Bridge.ts

```diff
@@ -218,32 +227,57 @@ describe('SimpleSchema2Bridge', () => {
       });
     });
 
-    it('works with options (array)', () => {
-      expect(bridge.getProps('s').transform('a')).toBe(1);
-      expect(bridge.getProps('s').transform('b')).toBe(2);
-      expect(bridge.getProps('s').allowedValues[0]).toBe('a');
-      expect(bridge.getProps('s').allowedValues[1]).toBe('b');
+    it('works with allowedValues (array)', () => {
+      expect(bridge.getProps('s').options[0]).toStrictEqual({
+        key: 'a',
+        label: 'a',
+        value: 'a',
+      });
+      expect(bridge.getProps('s').options[1]).toStrictEqual({
+        key: 'b',
+        label: 'b',
+        value: 'b',
+      });
+      expect(bridge.getProps('s').allowedValues).toBeUndefined();
     });
 
-    it('works with options (function)', () => {
-      expect(bridge.getProps('t').transform('a')).toBe(1);
-      expect(bridge.getProps('t').transform('b')).toBe(2);
-      expect(bridge.getProps('t').allowedValues[0]).toBe('a');
-      expect(bridge.getProps('t').allowedValues[1]).toBe('b');
+    it('works with allowedValues (function)', () => {
+      expect(bridge.getProps('t').options[0]).toStrictEqual({
+        key: 'a',
+        label: 'a',
+        value: 'a',
+      });
+      expect(bridge.getProps('t').options[1]).toStrictEqual({
+        key: 'b',
+        label: 'b',
+        value: 'b',
+      });
+      expect(bridge.getProps('t').allowedValues).toBeUndefined();
     });
 
-    it('works with options (object)', () => {
-      expect(bridge.getProps('r').transform('a')).toBe(1);
-      expect(bridge.getProps('r').transform('b')).toBe(2);
-      expect(bridge.getProps('r').allowedValues[0]).toBe('a');
-      expect(bridge.getProps('r').allowedValues[1]).toBe('b');
+    it('works with options (array)', () => {
+      expect(bridge.getProps('r').options[0]).toStrictEqual({
+        key: 'k1',
+        label: 'A',
+        value: 1,
+      });
+      expect(bridge.getProps('r').options[1]).toStrictEqual({
+        key: 'k2',
+        label: 'B',
+        value: 2,
+      });
     });
 
-    it('works with transform', () => {
-      expect(bridge.getProps('p')).toEqual({
-        label: 'P',
-        required: true,
-        transform: noopTransform,
+    it('works with options (function)', () => {
+      expect(bridge.getProps('rr').options[0]).toStrictEqual({
+        key: 'k1',
+        label: 'A',
+        value: 1,
+      });
+      expect(bridge.getProps('rr').options[1]).toStrictEqual({
+        key: 'k2',
+        label: 'B',
+        value: 2,
       });
```

Comment:
It'd be better to test the entire `options`, not only some properties. It'll also make the test smaller.